tests/src/rules/newline-after-import.js

```diff
@@ -349,5 +359,27 @@ ruleTester.run('newline-after-import', require('rules/newline-after-import'), {
       parserOptions: { sourceType: 'module' },
       parser: 'babel-eslint',
     },
+    {
+      code: `// issue 10042\nimport foo from 'foo';\n@SomeDecorator(foo)\nexport default class Test {}`,
```

Comment:
can these be reverted to use actual newlines instead of `\n`? it's much more readable.